scripts/publish.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -eu
+
+# For local development, in production, the environment will be set though GH actions and GH secrets
+if [ -f ".envrc" ]; then
+    echo "Loading .envrc"
+    # shellcheck disable=SC1091
+    . .envrc
+else
+    echo "No .envrc"
+fi
+
+echo "============================"
+echo "PAT: $PAT"
```

Comment:
Is this smart?

Reply:
It is not logged by GH actions and is useful for local development. If you believe that this should be removed strongly. I can remove it. 

![image](https://user-images.githubusercontent.com/746482/76523617-b4392900-648e-11ea-825c-8d205c5639af.png)

Reply:
Was not aware of automatic removal of secrets from logs.